Library/Formula/boost.rb

```diff
@@ -36,6 +36,10 @@ class Boost < Formula
     cause "Dropped arguments to functions when linking with boost"
   end
 
+  # Patch for odeint. Can (hopefully) be removed with the next release of Boost (1.58).
+  # See https://github.com/headmyshoulder/odeint-v2/issues/138
+  patch :DATA
```

Comment:
`odeint` needs to submit this to Boost if they haven't done so already. I don't think we can (or should) apply a patch to Boost for everyone until this has happened and Boost has merged it.